app/controllers/blog_controller.rb

```diff
@@ -9,5 +9,16 @@ def show
     def new
         @blog = Blog.new
     end
+    def create
+        @blog = Blog.create (blog_params)
+        if @blog.valid?
+            redirect_to blogs_path
+        
+        end
```

Comment:
Think you should have an else statement that handles unsuccessful create requests